src/Console/Command/MigrateCommand.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace MX\PhinxMigrations\Console\Command;
+
+use Magento\Setup\Console\Command\DbStatusCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\NullOutput;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Process;
+
+/**
+ * @package MX\PhinxMigrations
+ * @author  James Halsall <james.halsall@inviqa.com>
+ */
+class MigrateCommand extends Command
+{
+    protected function configure()
+    {
+        $this
+            ->setName('mx:migrate')
```

Comment:
maybe phinx:db:migrate or mx:phinx:migrate would be a better name (in case any other inviqa/mx module introduces new commands under "mx:"